packages/flutter/lib/src/cupertino/button.dart

```diff
@@ -47,6 +47,7 @@ class CupertinoButton extends StatefulWidget {
     this.color,
     this.minSize: 44.0,
     this.pressedOpacity: 0.1,
+    this.borderRadius: const BorderRadius.all(const Radius.circular(8.0)),
     @required this.onPressed,
   }) : assert(pressedOpacity >= 0.0 && pressedOpacity <= 1.0);
```

Comment:
looks like several of these arguments should be asserted-non-null and have a comment in the docs for the constructor that says they must not be null (see other widgets for examples of the style for these docs).